### PR TITLE
added update check

### PR DESCRIPTION
**Pull request by vintagesucks**
added update check

- compares the `version` value in terkelg/ramme/master/package.json against `app.getVersion()`
- displays dialog with option to download (opens [Releases](https://github.com/terkelg/ramme/releases) in browser) or ignore
- runs automatically on app start (with 10s delay to allow app to fully render, because [dialog.showMessageBox](http://electron.atom.io/docs/api/dialog/#dialogshowmessageboxbrowserwindow-options-callback) will block the process until the message box is closed)
- manual check for update via help menu

dialog
help menu

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
   "dependencies": {
     "electron-config": "^0.2.1",
     "electron-debug": "^1.0.1",
-    "element-ready": "^0.2.0"
+    "element-ready": "^0.2.0",
+    "request": "^2.74.0"
```
Comment (stephenkingsley) [inline package.json]: I do not agree ues `request`, I think you can code `nodejs http`

Comment (terkelg) [inline package.json]: I agree, It's a little overkill for one request

**Comment by terkelg** [comment]
Awesome! I'll look at it this weekend. Thanks

File: package.json (hunk 1 above)
Comment (vintagesucks) [inline package.json]: @terkelg Absolutely, removed that. :+1:

**Comment by terkelg** [comment]
Great, this is a really nice feature to have!

**Pull request merged**